### ISSUE TITLE
Error for command python manage.py makemigrations

### ISSUE THREAD

**Issue by adibalcan** [dancaron/Django-ORM#8]
I get tee following error when I run `python manage.py makemigrqtions`

`AttributeError: 'Settings' object has no attribute 'ROOT_URLCONF'`

**Comment by dancaron** [comment]
Can you post a larger code snippet?

**Comment by dancaron** [comment]
Is the following in your main.py?

```
import os
os.environ.setdefault("DJANGO_SETTINGS_MODULE", "settings")
import django
django.setup()
```

**Comment by adibalcan** [comment]
In main.py file I have a larger code, but I don't call the main.py I just run manage.py in order to make migration

**Comment by dancaron** [comment]
I'll see if I can reproduce the issue. If you have any relevant code (such as your models file), please post.

**Comment by adibalcan** [comment]
I use Django 1.9 if this helps

**Comment by dancaron** [comment]
Run 'python manage.py makemigrations db'

**Comment by dancaron** [comment]
Made some updates to the code. If you want, just follow the Quick Setup instructions again from scratch. Should work as expected now.

**Comment by adibalcan** [comment]
It's related with missing of PYTHONPATH envvar

**Comment by dancaron** [comment]
@adibalcan Everything working correctly? Can I close issue?

**Comment by adibalcan** [comment]
I still have the problem

**Comment by adibalcan** [comment]
I have added a ticket here: https://code.djangoproject.com/ticket/32857#ticket

**Comment by adibalcan** [comment]
I solved the issue by updating from Django 1.9 to Django 1.9.3
